src/Console/ModelsCommand.php

```diff
@@ -361,6 +361,7 @@ protected function getPropertiesFromTable($model)
         $schema = $model->getConnection()->getDoctrineSchemaManager($table);
         $databasePlatform = $schema->getDatabasePlatform();
         $databasePlatform->registerDoctrineTypeMapping('enum', 'string');
+        $databasePlatform->registerDoctrineTypeMapping('_int4', 'string');
```

Comment:
But isn't `_int4` rather an "array of int4", thus of type `array`?

Please try the following first if it fixes your problem: in your ide-helper config, add this:
```php
    'custom_db_types' => [
        'postgresql' => [
            '_int4' => 'array', //Map postgres "array of ints" type to array
        ],
    ],

```

Reply:
I had this same issue, and you solution worked perfectly.
Thanks a lot.